File: ulohy/13-trojuhelnik/reseni/main.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

rl.on("line", (line) => {
    const a = parseInt(line);

    for (let i = 0; i < a; i++) {
        for (let j = 0; j <= i; j++) {
            process.stdout.write("#");
        }
        process.stdout.write("\n");
    }

    rl.close();
});

File: ulohy/13-trojuhelnik/reseni/main.cs
using System;

internal class Program
{
    static void Main(string[] args)
    {
        int a = int.Parse(Console.ReadLine());

        for (int i = 0; i < a; i++)
        {
            for (int j = 0; j <= i; j++)
            {
                Console.Write('#');
            }
            Console.WriteLine();
        }
    }
}